brigade/core/task.py

```diff
@@ -61,10 +62,17 @@ def start(self, host, brigade):
             r = self.task(self, **self.params)
             if not isinstance(r, Result):
                 r = Result(host=host, result=r)
+
+        except BrigadeSubTaskError as e:
```

Comment:
I don't think we need this block, do we?

Reply:
There's a small difference as I don't save the traceback in the result key after BrigadeSubTaskError. Do you want to remove it?